app/models/skill.rb

```diff
@@ -0,0 +1,3 @@
+class Skill < ApplicationRecord
+	validates :name, presence: true, uniqueness: { case_sensitive: false }
```

Comment:
please change the setting of your editor to use 2 spaces instead of tab or 4 spaces.